test/reducers/user_training_status.spec.js

```diff
@@ -0,0 +1,23 @@
+import '../testHelper';
+import deepFreeze from 'deep-freeze';
+import user_training_status from '../../app/assets/javascripts/reducers/user_training_status';
+import { RECEIVE_USER_TRAINING_STATUS } from '../../app/assets/javascripts/constants';
+
+describe('user training status reducer', () => {
+  it('should return initial state when no action nor state is provided', () => {
+    const newState = user_training_status(undefined, { type: null });
+    expect(newState).to.be.an('object');
+  });
+
+  it('returns user training status with RECEIVE_USER_TRAINING_STATUS', () => {
+    const initialState = { title: 'title' };
```

Comment:
Here initial state is just an empty array. This particular reducer only is designed to hold a single array of data (for the user whose page we're on).